stix_shifter_modules/stix_bundle/configuration/config.json

```diff
@@ -13,6 +13,16 @@
         "help": {
             "type": "link",
             "default": "https://www.ibm.com/support/knowledgecenter/SSTDPP_1.1.0/docs/scp-core/data-sources-stix.html"
+        },
+        "options": {
+            "type": "fields",
+            "stix_validator": {
+                "type": "boolean",
+                "default": true,
+                "optional": true,
+                "hidden": true,
+                "previous": "connection.stix_validator"
```

Comment:
we have that value already, why did you insert it there: https://github.com/opencybersecurityalliance/stix-shifter/blob/develop/stix_shifter_modules/config.json#L45-L50

Reply:
so I needed to set the default value `"default": true,`

base config.json doesn't have any default parameter.

Reply:
> we have that value already, why did you insert it there: https://github.com/opencybersecurityalliance/stix-shifter/blob/develop/stix_shifter_modules/config.json#L45-L50

so I needed to set the default value "default": true,

base config.json doesn't have any default parameter.

Reply:
it should be enough to specify:
`        "options": {
            "stix_validator": {
                "default": true
            }
        }`

Reply:
and please set defalut:false in the base config.

Reply:
@yurii-klymenko updated